source/docs/software/mecanum-drive.rst

```diff
@@ -4,22 +4,22 @@ Programming Tutorial - Mecanum Drivetrain
 Mecanum Physics
 ---------------
 
-A very common choice for drivetrain in FTC is :term:`mecanum drive <Mecanum Wheel>`, but we have noticed that not a lot of people know how they work or how to properly program them.  :term:`Mecanum wheels <Mecanum Wheel>` have rollers at a 45° angle to the rest of the wheel.
+The :term:`mecanum drive <Mecanum Wheel>` is a very popular drive train in FTC. The mecanum drive enables holonomic movement. This means that the drivetrain is able move in all directions. Forward, backward, side to side, translating while rotating, etc. `Here is a neat video <https://www.youtube.com/watch?v=pP8ajNMx84k>`_ demonstrating such movement.
 
-Since these are in contact with the ground instead of something solid like in a :term:`traction wheel <Traction wheel>`, instead of the wheel creating a force parallel to the orientation of the wheel, it creates one 45° from parallel. Depending on how the wheels are driven, X or Y components of the force vectors can cancel which allows movement in any direction.
+:term:`Mecanum wheels <Mecanum Wheel>` have rollers at a 45° angle to the rest of the wheel. Since these are in contact with the ground instead of something solid like in a :term:`traction wheel <Traction wheel>`, instead of the wheel creating a force parallel to the orientation of the wheel, it creates one 45° from parallel. Depending on how the wheels are driven, X or Y components of the force vectors can cancel which allows movement in any direction.
 
 .. image:: images/mecanum-drive/mecanum-worms-eye-view.png
    :alt: Force diagram of a single mecanum wheel
 
 Using Vectoring to Create Omnidirectional Movement
 --------------------------------------------------
 
-As stated earlier, depending on how the wheels are driven, movement in any direction can be created. This is because with 4 wheels on a robot oriented in and X orientation, the vectors can be canceled such that movement is created in any direction. It is important to note that because of friction, perfect movement isn’t possible in every direction, so a :term:`mecanum drivetrain <Mecanum Wheel>` will be able to drive slightly faster forwards/backwards than any other directions.
+A standard mecanum drive configuration possesses 4 mecanum wheels oriented in an "X" shape. This means that the rollers are angled towards the center when looking at it from above. This configuration allows one to add up the force vectors generated by the offset rollers and derive movement in any direction. It is important to note that because of friction, perfect movement isn’t possible in every direction, so a :term:`mecanum drivetrain <Mecanum Wheel>` will be able to drive slightly faster forwards/backwards than any other directions. Combing translation and rotation will also result in slower movement.
```

Comment:
```suggestion
A standard mecanum drive configuration possesses 4 mecanum wheels oriented in an "X" shape. This means that the rollers are angled towards the center when looking at it from above. This configuration allows one to add up the force vectors generated by the offset rollers and derive movement in any direction. It is important to note that because of friction, perfect movement isn’t possible in every direction, so a :term:`mecanum drivetrain <Mecanum Wheel>` will be able to drive slightly faster forwards/backwards than any other directions. Combining translation and rotation will also result in slower movement.
```